tools/tplgtool2.py

```diff
@@ -842,13 +842,12 @@ def node_name_in_graph(self, node: Container) -> str:
         return self._node_name_in_graph_from_name(node["widget"]["name"])
 
     def node_sublabel(self, widget: Container) -> str:
-        match widget["widget"]["id"]:
-            case 'PGA':
-                sublabel = f'<BR ALIGN="CENTER"/><SUB>PGA: {widget["kcontrols"][0]["hdr"]["name"]}</SUB>'
-            case 'AIF_IN' | 'AIF_OUT' | 'DAI_IN' | 'DAI_OUT':
-                sublabel = f'<BR ALIGN="CENTER"/><SUB>{widget["widget"]["id"]}: {widget["widget"]["sname"]}</SUB>'
-            case _:
-                sublabel = ""
+        if widget["widget"]["id"] == 'PGA':
+             sublabel = f'<BR ALIGN="CENTER"/><SUB>PGA: {widget["kcontrols"][0]["hdr"]["name"]}</SUB>'
+        elif widget["widget"]["id"] in ('AIF_IN' , 'AIF_OUT' , 'DAI_IN' , 'DAI_OUT'):
```

Comment:
A tuple is a bit surprising, I think a list would be more... "pythonic"
```suggestion
        elif widget["widget"]["id"] in ['AIF_IN' , 'AIF_OUT' , 'DAI_IN' , 'DAI_OUT']:
```

Reply:
> A tuple is a bit surprising, I think a list would be more... "pythonic"

Ack, will fix in next update.